pkg/lua/Kconfig

```
@@ -0,0 +1,21 @@
+# Copyright (c) 2022 Inria
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config PACKAGE_LUA
+    bool "LUA language package"
+    depends on TEST_KCONFIG
+    depends on HAS_ARCH_32BIT
```

There are some blacklisted architectures as well
```suggestion
    depends on HAS_ARCH_32BIT
    depends on !HAS_ARCH_MIPS32R2
    depends on !HAS_ARCH_RISCV
```